tests/platforms/linux/appimage/test_run.py

```diff
@@ -10,6 +10,19 @@
 from briefcase.platforms.linux.appimage import LinuxAppImageRunCommand
 
 
+@pytest.mark.parametrize("host_os", ["Darwin"])
```

Comment:
Worth checking Windows here as well, to be explicit about the check.

Reply:
Added.